kong-0.5.0-1.rockspec

```diff
@@ -29,7 +29,8 @@ dependencies = {
 
   "luasocket ~> 2.0.2-5",
   "lrexlib-pcre ~> 2.7.2-1",
-  "lua-llthreads2 ~> 0.1.3-1"
+  "lua-llthreads2 ~> 0.1.3-1",
+  "sha1 ~> 0.5-1"
```

Comment:
I'm fine with the Luarocks module tbh it was just to make a distinction and keep the modules as lean as possible. Otherwise unused dependencies never get removed. 

> On Sep 18, 2015, at 5:18 PM, shashi ranjan <notifications@github.com> wrote:
> 
> In kong-0.5.0-1.rockspec:
> 
> > @@ -29,7 +29,8 @@ dependencies = {
> >  
> >    "luasocket ~> 2.0.2-5",
> >    "lrexlib-pcre ~> 2.7.2-1",
> > -  "lua-llthreads2 ~> 0.1.3-1"
> > +  "lua-llthreads2 ~> 0.1.3-1",
> > +  "sha1 ~> 0.5-1"
> Even i am using sha1 for test only otherwise i use ngx hmac-sha1function. Question is why to add library which is already available on luarocks?
> …
> —
> Reply to this email directly or view it on GitHub.
>